packages/teleport/src/cluster/components/Support/Support.tsx

```diff
@@ -0,0 +1,199 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import * as Icons from 'design/Icon';
+import styled from 'styled-components';
+import { Card } from 'design';
+import {
+  FeatureBox,
+  FeatureHeader,
+  FeatureHeaderTitle,
+} from 'teleport/components/Layout';
+import theme from 'design/theme/theme';
+import { useTeleport } from 'teleport/teleportContextProvider';
+
+const Support = () => {
+  const teleportCtx = useTeleport();
+  const cluster = teleportCtx.storeUser.state.cluster;
+
+  return (
+    <SupportBox>
+      <FeatureHeader>
+        <FeatureHeaderTitle>Help & Support</FeatureHeaderTitle>
+      </FeatureHeader>
+      <CardWrapper>
+        <MiniCard>
+          <h4>
+            <Icons.Stars />
+            Support
+          </h4>
+          <ul>
+            <Link
+              title="Create a Support Ticket"
+              url="https://github.com/gravitational/teleport/issues/new?labels=type%3A+question&template=question.md"
+            />
+            <Link
+              title="Ask the Community Questions"
+              url="https://community.gravitational.com/"
+            />
+            <Link
+              title="Request a New Feature"
+              url="https://github.com/gravitational/teleport/issues/new?labels=type%3A+feature+request&template=feature_request.md"
+            />
+            <Link title="Send Product Feedback" url="" />
+          </ul>
+        </MiniCard>
+        <MiniCard>
+          <h4>
+            <Icons.Stars />
+            Resources
+          </h4>
+          <ul>
+            <Link
+              title="Quickstart Guide"
+              url="https://gravitational.com/teleport/docs/quickstart/"
+            />
+            <Link
+              title="tsh User Guide"
+              url="https://gravitational.com/teleport/docs/user-manual/"
+            />
+            <Link
+              title="Admin Guide"
+              url="https://gravitational.com/teleport/docs/admin-guide/"
+            />
+            <Link
+              title="Download Page"
+              url="https://gravitational.com/teleport/download"
+            />
+            <Link
+              title="FAQ"
+              url="https://gravitational.com/teleport/docs/faq/"
+            />
+          </ul>
+        </MiniCard>
+        <MiniCard>
+          <h4>
+            <Icons.Stars />
+            Troubleshooting
+          </h4>
+          <ul>
```

Comment:
`<ul> <li>` is not welcome here as this is not a document published on internet. (there is literally 0 value of adding `<ul>` and extra `<li>`). Try using `<Box>` or `<Flex>` component instead.

`<h4>` needs to be repalced with `<Text typography="h4"/>`

Reply:
revised